<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Slides</class>
 <widget class="QMainWindow" name="Slides">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>742</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Slides</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>640</width>
     <height>480</height>
    </size>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>1024</x>
      <y>0</y>
      <width>256</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
outline: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>256</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
outline: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>256</x>
      <y>0</y>
      <width>768</width>
      <height>720</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border:none;
outline: none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1280</width>
      <height>720</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>Double-Click to Open Directory</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>pushButton_2</zorder>
   <zorder>pushButton</zorder>
   <zorder>pushButton_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QMenu" name="menuAuto_switch_image">
     <property name="title">
      <string>Auto-switch image</string>
     </property>
     <addaction name="action30_seconds"/>
     <addaction name="action1_minute"/>
     <addaction name="action2_minutes"/>
     <addaction name="action5_minutes"/>
     <addaction name="action10_minutes"/>
     <addaction name="separator"/>
     <addaction name="actionCustom"/>
     <addaction name="actionStop"/>
    </widget>
    <widget class="QMenu" name="menuAnimation">
     <property name="title">
      <string>Animation</string>
     </property>
     <addaction name="actionSlow"/>
     <addaction name="actionNormal"/>
     <addaction name="actionFast"/>
     <addaction name="separator"/>
     <addaction name="actionNone"/>
    </widget>
    <addaction name="actionNext_image"/>
    <addaction name="actionPrevious_image"/>
    <addaction name="separator"/>
    <addaction name="actionChange_image_randomly"/>
    <addaction name="separator"/>
    <addaction name="menuAuto_switch_image"/>
    <addaction name="actionShuffle"/>
    <addaction name="menuAnimation"/>
   </widget>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>Window</string>
    </property>
    <addaction name="actionFull_screen"/>
    <addaction name="actionMaximize"/>
    <addaction name="actionMinimize"/>
    <addaction name="separator"/>
    <addaction name="actionStandart_1280x720"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
   <addaction name="menuWindow"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionNext_image">
   <property name="text">
    <string>Next image</string>
   </property>
   <property name="shortcut">
    <string>Shift+Right</string>
   </property>
  </action>
  <action name="actionPrevious_image">
   <property name="text">
    <string>Previous image</string>
   </property>
   <property name="shortcut">
    <string>Shift+Left</string>
   </property>
  </action>
  <action name="action30_seconds">
   <property name="text">
    <string>30 seconds</string>
   </property>
  </action>
  <action name="action1_minute">
   <property name="text">
    <string>1 minute</string>
   </property>
  </action>
  <action name="action2_minutes">
   <property name="text">
    <string>2 minutes</string>
   </property>
  </action>
  <action name="action5_minutes">
   <property name="text">
    <string>5 minutes</string>
   </property>
  </action>
  <action name="action10_minutes">
   <property name="text">
    <string>10 minutes</string>
   </property>
  </action>
  <action name="actionCustom">
   <property name="text">
    <string>Custom...</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionShuffle">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Shuffle</string>
   </property>
  </action>
  <action name="actionChange_image_randomly">
   <property name="text">
    <string>Change image (randomly)</string>
   </property>
   <property name="shortcut">
    <string>Space</string>
   </property>
  </action>
  <action name="actionFull_screen">
   <property name="text">
    <string>Full-screen</string>
   </property>
   <property name="shortcut">
    <string>F</string>
   </property>
  </action>
  <action name="actionMaximize">
   <property name="text">
    <string>Maximize/Unmaximize</string>
   </property>
   <property name="shortcut">
    <string>M</string>
   </property>
  </action>
  <action name="actionMinimize">
   <property name="text">
    <string>Minimize</string>
   </property>
  </action>
  <action name="actionStandart_1280x720">
   <property name="text">
    <string>Standart (1280x720)</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionSlow">
   <property name="text">
    <string>Slow</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionFast">
   <property name="text">
    <string>Fast</string>
   </property>
  </action>
  <action name="actionNone">
   <property name="text">
    <string>None</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
